index.html

```diff
@@ -5,25 +5,17 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=yes">
     <title>Rapido</title>
-    <meta name="description" content="Rapido: semantic HTML elements for your writings.">
+    <meta name="description" content="Rapido.css: semantic HTML for your writings.">
     <meta name="robots" content="index,follow">
 
     <link rel="icon" type="image/png" href="favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="favicon-16x16.png" sizes="16x16" />
 
     <meta property="og:url" content="https://nextbitlabs.github.io/Rapido/">
-    <meta property="og:type" content="website">
-    <meta property="og:title" content="Rapido">
-    <!-- <meta property="og:image" content="https://nextbitlabs.github.io/Rapido/images/intro.png"> -->
-    <meta property="og:description" content="Rapido: semantic HTML elements for your writings.">
-    <!-- <meta property="og:site_name" content="nextbit.it"> -->
-    <meta property="og:locale" content="en_US">
-
-    <meta name="twitter:card" content="Rapido: semantic HTML elements for your writings.">
-    <meta name="twitter:url" content="https://nextbitlabs.github.io/Rapido/">
-    <meta name="twitter:title" content="Rapido">
-    <meta name="twitter:description" content="Rapido: semantic HTML elements for your writings.">
-    <!-- <meta name="twitter:image" content="https://nextbitlabs.github.io/Rapido/images/intro.png"> -->
+    <meta property="og:title" content="Rapido.css: semantic HTML for your writings.">
+    <meta property="og:image" content="https://nextbitlabs.github.io/Rapido/media/card.png">
+    <meta property="og:description" content="Rapido provides a default style for a set of semantic HTML elements that you can use to compose your essays. Just write with semantic HTML, without the need to remember any CSS class or JS command.">
+    <meta name="twitter:card" content="summary_large_image">
```

Comment:
Isn't the url tag necessary?

Reply:
There is, it is at line 14 `<meta property="og:url" content="https://nextbitlabs.github.io/Rapido/">`.